### PR TITLE
[feat-4983]: Add version number fe to docker hub workflow

### PR DESCRIPTION
**Pull request by OscarBakker**
So docker can pick it up and provide it to the fe, showing to in the console.

Ticket: [SIG-4983](https://gemeente-amsterdam.atlassian.net/browse/SIG-4983)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: .github/workflows/build_and_push_to_docker_hub.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
 
       - name: Build the release Docker image
         run: |
-          docker build -t signalen/frontend:${{ github.event.release.tag_name }} .
+          docker build -t signalen/frontend:${{ github.event.release.tag_name }} |
```
Comment (bartjkdp) [inline .github/workflows/build_and_push_to_docker_hub.yml:23]:
```suggestion
          docker build -t signalen/frontend:${{ github.event.release.tag_name }}
```

File: .github/workflows/build_and_push_to_docker_hub.yml [hunk 2]
```diff
@@ -20,7 +20,8 @@ jobs:
...
+          --build-arg FRONTEND_TAG=${{ github.event.release.tag_name }}.
```
Comment (bartjkdp) [inline .github/workflows/build_and_push_to_docker_hub.yml:24]:
```suggestion
          --build-arg FRONTEND_TAG=${{ github.event.release.tag_name }} .
```

File: .github/workflows/build_and_push_to_docker_hub.yml (hunk 1 above)
Comment (OscarBakker) [reply .github/workflows/build_and_push_to_docker_hub.yml:23]: aaah you start with pipe!

File: .github/workflows/build_and_push_to_docker_hub.yml [hunk 3]
```diff
@@ -20,7 +20,8 @@ jobs:
 
       - name: Build the release Docker image
         run: |
-          docker build -t signalen/frontend:${{ github.event.release.tag_name }} .
+          docker build -t signalen/frontend:${{ github.event.release.tag_name }}
```
Comment (bartjkdp) [inline .github/workflows/build_and_push_to_docker_hub.yml:23]:
```suggestion
          docker build -t signalen/frontend:${{ github.event.release.tag_name }} \
```

**Review by bartjkdp — COMMENTED** [review]
I am not sure, but I think this requires an additional `\`. What do you think? I used it [here](https://github.com/Signalen/frontend/blob/develop/.github/workflows/review_deploy.yml#L69) for multi-line commands as well.

**Review by vdegraaf — COMMENTED** [review]
Where can I find the tag? The console stills says "dummy": 
Screenshot 2023-02-21 at 16 35 03

**Review by bartjkdp — APPROVED** [review]

**Pull request merged**